### PR TITLE
Pic 1181 - Getting "Undefined" when no forename is in data

### PR DESCRIPTION
**Pull request by sarahyjja**

### CODE REVIEW

File: mappings/case-list-full.json [hunk 1]
```diff
@@ -281,6 +279,11 @@
           ],
           "defendantSex": "M",
           "defendantName": "MR Webb Mitchell",
+          "name": {
+            "title": "Mister",
+           "forename1": "webster",
```
Comment (katie-cawthorne) [inline mappings/case-list-full.json:284]: We could probably do with an example in the mock data where there is no `forename1` or `surname` just to show that it defaults to `defendantName`.

Comment (sarahyjja) [reply mappings/case-list-full.json:283]: Modification has been done to `MISS FELICIA Villarreality`

**Review by sarahyjja — COMMENTED** [review]
Change made on `MISS FELICIA Villarreality`

**Review by katie-cawthorne — APPROVED** [review]
LGTM ✨

**Pull request merged**